src/lib/launchDarkly.ts

```diff
@@ -27,6 +27,20 @@ export type LDFlagsValue = JSONValue;
 /** Flags for LaunchDarkly */
 export type LDFlags = Record<string, LDFlagsValue>;
 
+/**
+ * Checks if the LaunchDarkly client is initialized.
+ * @param {LDClient.LDClient | undefined} client - The LaunchDarkly client.
+ * @throws {Error} If the client is not initialized.
+ */
+function isClientInitialized(
+	client: LDClient.LDClient | undefined
+): asserts client is LDClient.LDClient {
+	console.log('isClientInitialized', client);
```

Comment:
Not sure if this should be logged every time or follow the same pattern of `debugLog`